src/json-viewer.js

```diff
@@ -0,0 +1,318 @@
+var jsonObj = {};
+var filePath = "data-example/NCT001.json";
+loadJSON(filePath, function(response) {
+    // Parse JSON string into object. You can assign you own json data to jsonObj directly if you don't need to read file.
+    jsonObj = JSON.parse(response);
+    var jsonViewer = new JSONViewer();
+    document.querySelector("#json").appendChild(jsonViewer.getContainer());
+    jsonViewer.showJSON(jsonObj);
+});
+
+function loadJSON(file, callback) {
+    var xhr = new XMLHttpRequest();
+    xhr.overrideMimeType('application/json');
+    xhr.open('GET', file, true);
+    xhr.onreadystatechange = function() {
+        if (xhr.readyState === 4 && xhr.status === 200) {
+            callback(xhr.responseText);
+        }
+    };
+    xhr.send(null);
+}
+
+/**
+ * JSONViewer - by Roman Makudera 2016 (c) MIT licence.
+ */
+JSONViewer = (function() {
```

Comment:
I think at the end of the day, this file will be used separately. Line 1-21 is just giving example how to use this lib, so we probably should just separate it to a lib.

Reply:
Ok, I'll separate the file.